azure-pipelines.yml

```diff
@@ -13,7 +13,7 @@ pr:
     - '*'
 
 variables:
-- template: .azure-pipelines/templates/variables.yml
+- template: ${{ variables.Pipeline.Workspace }}/.azure-pipelines/templates/variables.yml
```

Comment:
The template path uses relative path by default.
Actually there is no need to modify the template path in azure-pipelines.yml and azure-pipelines-full-tests.yml.
Just to keep things consistent.